app/views/spree/components/products/_products.html.erb

```diff
@@ -9,7 +9,8 @@
   <% end %>
 <% end %>
 
-<% if products.empty? %>
+<% no_products_found = products.empty? %>
```

Comment:
What about moving this on top of the file where other variables are set?

Reply:
Sure 👍